lib/node_modules/@stdlib/string/reverse/test/test.js

```diff
@@ -116,3 +200,82 @@ tape( 'the function reverses a string', function test( t ) {
 	}
 	t.end();
 });
+
+tape( 'the function reverses the first string (mode=grapheme)', function test( t ) {
+	var opts;
+	var out;
+
+	opts = {
+		'mode': 'grapheme'
+	};
+
+	out = reverseString( 'hello world', opts );
+	t.strictEqual( out, 'dlrow olleh', 'returns expected value' );
+
+	out = reverseString( '*!!!', opts );
+	t.strictEqual( out, '!!!*', 'returns expected value' );
+
+	out = reverseString( 'अनुच्छेद', opts );
+	t.strictEqual( out, 'दछेच्नुअ', 'returns expected value' );
+
+	out = reverseString( '六书/六書', opts );
+	t.strictEqual( out, '書六/书六', 'returns expected value' );
+
+	out = reverseString( '🌷', opts );
+	t.strictEqual( out, '🌷', 'returns expected value' );
+
+	out = reverseString( '👉🏿', opts );
+	t.strictEqual( out, '🏿👉', 'returns expected value' );
```

Comment:
This looks off to me. I'd expect that, similar to L225, the input string equals the output string. Maybe this stems from using `prevGraphemeClusterBreak`?